docs/src/development/environment-setup.md

```diff
@@ -13,39 +14,37 @@ If you already have Rust installed make sure to upgrade it by running
 $ rustup update
 ```
 
-To verify that project was cloned and set up correctly, you can run
+### Scarb
+You can read more about installing scarb [here](https://docs.swmansion.com/scarb/download.html)
 
-```shell
-$ cargo check
-```
+Please make sure you're using scarb installed via asdf - otherwise some tests may fail.
+> To verify, run:
+> 
+> ```shell
+> $ which scarb
+> ```
+> the result of which should be:
+> ```shell
+> $HOME/.asdf/shims/scarb
+> ```
+> 
+> If you previously installed scarb using official installer, you may need to remove this installation or modify your PATH to make sure asdf installed one is always used.
+
+### Starknet Devnet
+To install it run `./scripts/install_devnet.sh`
+
+### Universal sierra compiler
+Install the latest [universal-sierra-compiler](https://github.com/software-mansion/universal-sierra-compiler) version.
 
-## Running Tests
 
-> 📝 **Note**
-> 
-> Make sure you run `./scripts/install_devnet.sh`
-> and then set [Scarb](https://docs.swmansion.com/scarb/) version 
-> [compatible](https://github.com/foundry-rs/starknet-foundry/releases) with both `snforge` and `sncast`
-> and use the newest [universal-sierra-compiler](https://github.com/software-mansion/universal-sierra-compiler)
-> after setting up the development environment, otherwise the tests will fail.
 
+## Running Tests
```

Comment:
I was thinking about adding info about those two tests:
-`e2e::running::init_new_project_test`
-`e2e::running::simple_package_with_git_dependency`
since they fail locally if the branch you're working on hasn't been pushed to the remote yet.

Did something like that [here](https://github.com/foundry-rs/starknet-foundry/pull/2006/files#diff-32059288e9fa43c9310b25828adab2a8ba883a299e6cc7acb0df582f2de5ee85R59), what do you think?

Reply:
Great idea, I am not adding it here as I assume you will merge into those changes.